spec/defines/openvpn_server_spec.rb

```diff
@@ -166,7 +166,7 @@
         'key_cn'          => 'yolo',
         'key_name'        => 'burp',
         'key_ou'          => 'NSA',
-        'verb'            => 'mute',
```

Comment:
@luxflux do you know why verb is 'mute' here? It seems like `mute` is a separate param, and the docs suggest that `verb` should be an int as in the client?

Reply:
Yeah, this seems to be wrong. Let's use an integer...
From https://openvpn.net/index.php/open-source/documentation/manuals/65-openvpn-20x-manpage.html:
> --verb n
  Set output verbosity to n (default=1). Each level shows all info from the previous levels. Level 3 is recommended if you want a good summary of what's happening without being swamped by output.
  0 -- No output except fatal errors. 
  1 to 4 -- Normal usage range. 
  5 -- Output R and W characters to the console for each packet read and write, uppercase is used for TCP/UDP packets and lowercase is used for TUN/TAP packets. 
  6 to 11 -- Debug info range (see errlevel.h for additional information on debug levels).

I think we need we need to update this line as well: https://github.com/wyardley/puppet-openvpn/blob/7b4d56a6d048956dd8f706449f2158966745b68f/spec/defines/openvpn_server_spec.rb#L223